integration/live_update_two_images_one_manifest_test.go

```diff
@@ -0,0 +1,92 @@
+//+build integration
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	sparkleUrl = "http://localhost:8100"
+	tadaUrl    = "http://localhost:8101"
```

Comment:
go style is tadaURL, not tadaUrl
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Whoops, you're right!